gradle-language-server/build.gradle

```diff
@@ -23,7 +23,7 @@ application {
 dependencies {
   implementation "org.eclipse.lsp4j:org.eclipse.lsp4j:0.12.0"
   implementation "org.eclipse.lsp4j:org.eclipse.lsp4j.jsonrpc:0.12.0"
-  implementation "org.codehaus.groovy:groovy-eclipse-batch:4.0.15-03"
+  implementation "org.codehaus.groovy:groovy-eclipse-batch:4.0.16-03"
```

Comment:
I bumped this dependency just as we did on #1419, which I think helps with Java 21 compatibility, but I'm not sure because I couldn't find much documentation about it. I can rever this if it's not necessary at this point 😁

Reply:
Should be fine.